sdk/search/Azure.Search.Documents/src/SearchServiceClient.cs

```diff
@@ -172,71 +184,227 @@ public virtual SearchIndexClient GetSearchIndexClient(string indexName)
-            Protocol.GetServiceStatistics(
+            ServiceClient.GetServiceStatistics(
                 options?.ClientRequestId,
                 cancellationToken);
 
         /// <summary>
-        /// Gets service level statistics for a Search Service.
-        ///
+        /// <para>
+        /// Gets service level statistics for a Search service.
+        /// </para>
+        /// <para>
         /// This operation returns the number and type of objects in your
         /// service, the maximum allowed for each object type given the service
         /// tier, actual and maximum storage, and other limits that vary by
         /// tier.  This request pulls information from the service so that you
         /// don't have to look up or calculate service limits.
-        ///
+        /// </para>
+        /// <para>
         /// Statistics on document count and storage size are collected every
         /// few minutes, not in real time.  Therefore, the statistics returned
         /// by this API may not reflect changes caused by recent indexing
         /// operations.
+        /// </para>
         /// </summary>
         /// <param name="options">
-        /// Options to customize the operation's behavior.
+        /// Optional <see cref="SearchRequestOptions"/> to customize the operation's behavior.
         /// </param>
         /// <param name="cancellationToken">
         /// Optional <see cref="CancellationToken"/> to propagate notifications
         /// that the operation should be canceled.
         /// </param>
         /// <returns>The service level statistics.</returns>
         /// <exception cref="RequestFailedException">
-        /// Thrown when a failure is returned by the Search Service.
+        /// Thrown when a failure is returned by the Search service.
+        /// </exception>
+        [ForwardsClientCalls]
+        public virtual async Task<Response<SearchServiceStatistics>> GetServiceStatisticsAsync(
+            SearchRequestOptions options = null,
+            CancellationToken cancellationToken = default) =>
+            await ServiceClient.GetServiceStatisticsAsync(
+                options?.ClientRequestId,
+                cancellationToken)
+                .ConfigureAwait(false);
+        #endregion
+
+        #region Index operations
+        /// <summary>
+        /// Creates a new search index in the Search service.
+        /// </summary>
+        /// <param name="index">
+        /// The <see cref="SearchIndex"/> to create.
+        /// </param>
+        /// <param name="options">
+        /// Optional <see cref="SearchRequestOptions"/> to customize the operation's behavior.
+        /// </param>
+        /// <param name="cancellationToken">
+        /// Optional <see cref="CancellationToken"/> to propagate notifications
+        /// that the operation should be canceled.
+        /// </param>
+        /// <returns>
+        /// The <see cref="SearchIndex"/> that was created.
```

Comment:
How does this differ from what we passed in?

Reply:
Some fields come back with their default values in the response that might not have been in the request. Track 1 did it this way, as does Java track 2.

Reply:
It'd be great to add that to the doc comment.